lib/routes.php

```diff
@@ -218,6 +218,7 @@
 $app->get('/library/usage/layouts/:id', '\Xibo\Controller\Library:usageLayouts')->name('library.usage.layouts');
 $app->get('/library/download/:id(/:type)', '\Xibo\Controller\Library:download')->name('library.download');
 $app->post('/library', '\Xibo\Controller\Library:add')->name('library.add');
+$app->post('/library/uploadURL', '\Xibo\Controller\Library:uploadFromURL')->name('library.uploadFromURL');
```

Comment:
I think this should be camel case to fit without convention elsewhere? `uploadUrl`